src/main/java/ch/njol/skript/lang/Conditional.java

```diff
@@ -45,6 +45,10 @@ public Conditional(final Condition cond, final SectionNode node) {
 		this.cond = cond;
 	}
 	
+	public Conditional(final Condition cond) {
```

Comment:
instead of making a new constructor why not just call the existing one with `Conditional(cond, null)`

Reply:
Ok, I fixed that up and removed that extra constructor

Reply:
Ok, I had to bring this back.
Using 'null' in the contractor caused every single condition to fail regardless if it was actually parsable or not.